integration-tests/src/test/java/oracle/weblogic/kubernetes/ItT3Channel.java

```diff
@@ -290,6 +293,28 @@ public void testAdminServerT3Channel() {
 
     //verify admin server accessibility and the health of cluster members
     verifyMemberHealth(adminServerPodName, managedServerNames, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
+
+    // Since the ListenPort is not set explicitly on ServerTemplate 
+    // in wlst offline script wlst-create-domain-onpv.py, the default 
+    // ListenPort on each manged server is set to 7100. This can be confirmed
+    // by intorospecting the corresponding cluster service port (default)
+    int servicePort = assertDoesNotThrow(()
+        -> getServicePort(domainNamespace, 
+              domainUid + "-cluster-" + clusterName, "default"),
+              "Getting Cluster Service default port failed");
+    assertEquals(7100, servicePort, "Default Service Port is not set to 7100");
```

Comment:
can you create a constant for 7100 here?

Reply:
done